example/android/app/build.gradle

```diff
@@ -129,7 +126,7 @@ android {
         variant.outputs.each { output ->
             // For each separate APK per architecture, set a unique version code as described here:
             // http://tools.android.com/tech-docs/new-build-system/user-guide/apk-splits
-            def versionCodes = ["armeabi-v7a":1, "x86":2]
+            def versionCodes = ["armeabi-v7a": 1, "x86": 2, "arm64-v8a": 3, "x86-64": 4]
```

Comment:
@kdawgwilk I think x86-64 was a typo in the React Native repo, as they changed it in todays .59.1 bugfix version to x86_64. Can we change the two instances of this or is there a reason we wouldn't want to change it?
See https://github.com/facebook/react-native/commit/4b996da470b43f97fd0426b54bda739d7717fb28 for related change.